### DISCUSSION TITLE
请问工作人员角色如何设置权限才有测试实例连通性的权限？

### DISCUSSION THREAD

**Discussion by carlors** [hhyo/Archery#1818] [Q&A]
工作人员录入数据库连接地址、密码等信息后，点击“测试”连接报没有权限，查看权限列表，没看到对应的权限，求问需要赋予工作人员角色什么权限才能测试数据库实例连接？

**Comment by hhyo** [comment]
当前没有单独设置这个权限，仅开放给管理员
